containers/ecr-viewer/src/app/data/blobStorage/gcsClient.ts

```diff
@@ -0,0 +1,40 @@
+import { Storage } from "@google-cloud/storage";
+
+/**
+ * Connect to the google cloud storage bucket.
+ * @returns The google cloud storage bucket.
+ */
+export const gcsClient = () => {
+  if (process.env.GCS_BUCKET_NAME) {
```

Comment:
why check of the bucket and not the source?

Reply:
Good point! I'm not sure why I didn't do that in the first place with these health checks. I'll open another PR to address those other ones!